lib/docker-sync/dependencies/fswatch.rb

```diff
@@ -0,0 +1,14 @@
+module DockerSync
+  module Dependencies
+    module Fswatch
+      def self.available?
+        return @available if defined? @available
```

Comment:
We can but that would make it dependant / coupled to `Environment`, which is something I'm trying to avoid. Do we really need it? (`find_executable0` is not an expensive call, if that's what you'd like to prevent)